data/brands/shop/alcohol.json

```diff
@@ -964,12 +964,8 @@
       "matchNames": ["красное и белое"],
       "tags": {
         "brand": "Красное&Белое",
-        "brand:en": "Red&White",
```

Comment:
You can look at other Russian names in this file and see that they generally don't have transliterations, for example https://github.com/osmlab/name-suggestion-index/blob/1e1b3ae3c2d569800c0105b428a6a367f0edd44d/data/brands/shop/alcohol.json#L950

The exception to that seems to be https://github.com/osmlab/name-suggestion-index/blob/1e1b3ae3c2d569800c0105b428a6a367f0edd44d/data/brands/shop/alcohol.json#L1021 (because it's not a Russian word?)